racetrack_client/racetrack_client/client/delete.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+from racetrack_client.client_config.alias import resolve_lifecycle_url
+from racetrack_client.client_config.auth import get_user_auth
+from racetrack_client.client_config.io import load_client_config
+from racetrack_client.log.logs import get_logger
+from racetrack_client.manifest.validate import load_validated_manifest
+from racetrack_client.utils.auth import get_auth_request_headers
+from racetrack_client.utils.request import parse_response, Requests
+
+
+logger = get_logger(__name__)
+
+
+def delete_job(workdir: str, lifecycle_url: Optional[str], version: Optional[str]):
```

Comment:
I think you didn't merge the latest changes from master. It's been moved to `manage.py` file